manuscript/11_data_classes.md

```diff
@@ -29,7 +29,7 @@ fun main() {
 
 > Prawdę mówiąc, `Any` jest reprezentowane jako klasa, ale powinno być traktowane jako typ. Weź pod uwagę fakt, że `Any` jest także nadrzędnym typem wszystkich interfejsów, mimo że interfejsy nie mogą dziedziczyć po klasach.
 
-Domyślne implementacje `equals`, `hashCode` i `toString` są mocno oparte na adresie obiektu w pamięci. Metoda `equals` zwraca `true` tylko wtedy, gdy adres obu obiektów jest taki sam, co oznacza, że po obu stronach jest ten sam obiekt. Metoda `hashCode` zamienia ten adres na liczbę. `toString` generuje string, który zaczyna się od nazwy klasy, a następnie znaku małpy "@", a potem skrótu adresu obiektu w notacji szesnastkowej.
+Domyślne implementacje `equals`, `hashCode` i `toString` są oparte na tak zwanym kodzie hash tożsamości (`System.identityHashCode()`). Metoda `equals` zwraca `true` tylko wtedy, gdy dla obu obiektów jest taki sam, co oznacza, że po obu stronach jest ten sam obiekt. Metoda `hashCode` zwraca `identityHashCode`. `toString` generuje string, który zaczyna się od nazwy klasy, a następnie znaku małpy "@", a potem `identityHashCode` w notacji szesnastkowej.
```

Comment:
Pierwszo słyszę o "kodzie hash tożsamości" i nie wydaje mi się to upraszczać tematu, pozostałbym przy "mocno oparte na adresie obiektu w pamięci."